### PR TITLE
Add functionality SetIdentitiMailFromDomain, GetIdentityMailFromDomai…

### PR DESCRIPTION
**Pull request by usmangani1**
Add functionality SetIdentitiMailFromDomain, GetIdentityMailFromDomainAttributes

https://github.com/localstack/localstack/issues/4276

### CODE REVIEW

File: moto/ses/models.py [hunk 1]
```diff
@@ -414,5 +416,24 @@ def create_receipt_rule(self, rule_set_name, rule):
         rule_set.append(rule)
         self.receipt_rule_set[rule_set_name] = rule_set
 
+    def set_identity_mail_from_domain(
+        self, identity, mail_from_domain=None, behavior_on_mx_failure=None
+    ):
+        if identity not in self.domains:
+            raise IdentityDoesNotExist(identity)
```
Comment (bblommers) [inline moto/ses/models.py:423]: This will need a unit test

File: tests/test_ses/test_ses_boto3.py [hunk 2]
```diff
@@ -614,3 +614,19 @@ def test_domains_are_case_insensitive():
...
+    conn = boto3.client("ses", region_name="us-east-1")
+    conn.verify_domain_dkim(Domain="domain1.com")
+
+    conn.set_identity_mail_from_domain(
+        Identity="domain1.com",
+        MailFromDomain="boundes.example.com",
+        BehaviorOnMXFailure="UseDefaultValue",
+    )
+
+    response = conn.get_identity_mail_from_domain_attributes(Identites=["domain1.com"])
```
Comment (bblommers) [inline tests/test_ses/test_ses_boto3.py:630]: Typo - 'Identities'

File: tests/test_ses/test_ses_boto3.py [hunk 3]
```diff
@@ -614,3 +614,19 @@ def test_domains_are_case_insensitive():
...
+    print(response)
+    assert "MailFromDomainAttributes" in response
```
Comment (bblommers) [inline tests/test_ses/test_ses_boto3.py:632]: We know exactly the response format, so this should be more explicit - i.e., create the exact same dict and assert equality

File: tests/test_ses/test_ses_boto3.py (hunk 2 above)
Comment (bblommers) [reply tests/test_ses/test_ses_boto3.py:630]: Can we get a few more calls here, and verify the correct behaviour response for various invocations:
 - get_identity_mail_from_domain_attributes before setting anything
 - get_identity_mail_from_domain_attributes(Identities=None) (after calling set_identity_mail_from_domain)
 - get_identity_mail_from_domain_attributes(Identities=[]) (after calling set_identity_mail_from_domain)

**Review by bblommers — CHANGES_REQUESTED** [review]

**Comment by krewenki** [comment]
@usmangani1 Have you made any further progress on this?  I'm hitting issues related to this in localstack:

```
Aws\Ses\Exception\SesException: Error executing "GetIdentityDkimAttributes"
```

**Pull request closed without merging**